### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by kswang2400**
PiperOrigin-RevId: 399233865

### CODE REVIEW

File: ontology/yaml/resources/GATEWAYS/entity_types/GW.yaml [hunk 1]
```diff
@@ -1209,4 +1209,4 @@ LINX:
   - supply_water_temperature_setpoint_5
   - outside_air_relative_humidity_sensor
   - heating_supply_water_temperature_setpoint
-
```
Comment (charbull) [inline ontology/yaml/resources/GATEWAYS/entity_types/GW.yaml:1212]: revert? otherwise it will keep bouncing?

Comment (kswang2400) [reply ontology/yaml/resources/GATEWAYS/entity_types/GW.yaml:1212]: we reverted on the cl, this is going to happen regardless :(

File: ontology/yaml/resources/HVAC/entity_types/PHRED_GENERATED.yaml [hunk 2]
```diff
@@ -0,0 +1,5795 @@
+# Generated @ 2021-09-21T21:15:41.212438Z
+CH_CHWS_PMP_US_SVL_TC4_1:
```
Comment (charbull) [inline ontology/yaml/resources/HVAC/entity_types/PHRED_GENERATED.yaml:2]: can you please remove this ?

File: ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -67,9 +67,8 @@ SCZ:
   id: "2665719349737684992"
   is_abstract: true
   description: "Scene control for a device and zone."
-  opt_uses:
```
Comment (charbull) [inline ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml:70]: is this change legitimate?

File: ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml [hunk 4]
```diff
@@ -21,78 +21,3 @@ LGRP_DC:
   implements:
     - LGRP
     - DC
-
```
Comment (charbull) [inline ontology/yaml/resources/LIGHTING/entity_types/LGRP.yaml:24]: these need to stay

**Review by charbull — CHANGES_REQUESTED** [review]
some minor details to pay attention to

**Pull request closed without merging**